core/src/data/value/binary_op/i8.rs

```diff
@@ -284,14 +341,15 @@ mod tests {
         let type_min: i8 = i8::MIN;
         let type_maxi64: i64 = type_max.into();
         let type_mini64: i64 = type_min.into();
+        let type_maxi128: i128 = type_max.into();
+        let type_mini128: i128 = type_min.into();
```

Comment:
maybe this too

Reply:
Like the above ones, it would be also just good not to use variables, explicit use of `i8::MIN` would be straightforward enough.